### PR TITLE
[WIP] Multi snapshot TC refactoring

### PR DESCRIPTION
**Pull request by Avilir**
This PR is a re-factoring of the Multi-Snapshots TC to use only one script and not 2 scripts,
and adding a verification of the teardown process to make sure all data was deleted.

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py [hunk 1]
```diff
@@ -1,39 +1,388 @@
...
+
 
 @performance
 @skipif_ocp_version("<4.6")
 @skipif_ocs_version("<4.6")
-class TestPvcMultiSnapshotPerformance(E2ETest):
+class TestPvcMultiSnapshotPerformance(PASTest):
     """
     Tests to measure PVC snapshots creation performance & scale
     The test is trying to to take the maximum number of snapshot for one PVC
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py:46]: small spelling fix - maximal , not maximum

File: tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py [hunk 2]
```diff
@@ -1,39 +1,388 @@
...
+            Switch to the default namespace
+            Delete the tested namespace
+
+        """
+        log.info("Cleanup the test environment")
+
+        try:
+            vs_list = self.snapshot.get(all_namespaces=True)["items"]
+        except Exception as err:
+            log.error(f"Can not get the list of snapshot : {err}")
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py:100]: spelling fix: "Cannot get the list of snapshots"

File: tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py [hunk 3]
```diff
@@ -1,39 +1,388 @@
...
+            f"Start creation time is : {st}, End creation time is : {et}"
+            f" and Total creation time is {results}"
+        )
+
+        return results
+
+    def run(self):
+        """
+        Running the test
+            foreach snapshot : write data on the pod and take snapshot
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py:351]: for each - pleae add space

**Review by ypersky1980 — APPROVED** [review]

**Comment by ypersky1980** [comment]
Left some minor comments.

**Review by ypersky1980 — APPROVED** [review]

File: ocs_ci/ocs/perftests.py [hunk 4]
```diff
@@ -66,6 +75,46 @@ def teardown(self):
         if hasattr(self, "operator"):
             self.operator.cleanup()
 
+        now_data = self.get_cephfs_data()
+        # Wait 1 minutes for the backend deletion actually start.
+        log.info("Waiting for Ceph to finish cleaning up")
+        time.sleep(60)
+
+        # Quarry the storage usage every 2 Min. if no difference between two
```
Comment (pintojoy) [inline ocs_ci/ocs/perftests.py:83]: spelling fix: Query the storage ...

**Review by ypersky1980 — APPROVED** [review]
LGTM

**Review by keemano — CHANGES_REQUESTED** [review]
`single comment`

**Comment by Avilir** [comment]
Closing since new PR to fix issues in this one are WIP

**Pull request closed without merging**